packages/template-retail-react-app/translations/en-US.json

```diff
@@ -384,6 +384,9 @@
     "defaultMessage": "This 4-digit code can be found on the front of your card.",
     "description": "American Express security code help text"
   },
+  "credit_card_fields.tool_tip.security_code_aria_label": {
```

Comment:
Make sure this one go to en-GB too, other we will see missing translation in the console while running the app